opentreemap/treemap/templates/treemap/partials/user_favorite.html

```diff
@@ -0,0 +1,13 @@
+{% load util %}
+
+<a href="{{ favorite.map_feature|detail_link }}">
+    <div>
+        {% if favorite.photo %}
+            <img src="{{ favorite.photo }}" />
+        {% else %}
+            <div class="no-photo"><i class="icon-star"></i></div>
+        {% endif %}
+        <h4 class="common-name">{{ favorite.title }}</h4>
+        <h5>{{favorite.address}} - {{favorite.instance.name}}</h5>
```

Comment:
To me it would look better to use an em dash (`&mdash;`) or parentheses for the instance name, e.g.:

4910 S Boyle Ave, Vernon, 90058 - TreeMapLA

4910 S Boyle Ave, Vernon, 90058 — TreeMapLA

4910 S Boyle Ave, Vernon, 90058 (TreeMapLA)